testbed/lib/main.dart

```diff
@@ -323,17 +348,13 @@ class FileChooserTestWidget extends StatelessWidget {
               initialDirectory =
                   (await getApplicationDocumentsDirectory()).path;
             }
-            file_chooser
-                .showOpenPanel(
-                    allowsMultipleSelection: true,
-                    initialDirectory: initialDirectory)
-                .then(
-              (result) {
-                Scaffold.of(context).showSnackBar(SnackBar(
-                    content: Text(_resultTextForFileChooserOperation(
-                        _FileChooserType.open, result))));
-              },
-            );
+            final result = await file_chooser.showOpenPanel(
```

Comment:
I'm happy to take improvements to the Testbed code in general, but this (and the change above) is unrelated to the PR. I would much rather you just send a PR containing just the miscellaneous cleanup, which could be trivially reviewed and landed separately from the behavior-altering change that this PR is intended to be making.

Reply:
Reverting. Due to bad life choices (creating this PR in the master branch of my fork), these fix ups will have to wait until a follow up PR.